src/portfolios/portfolio/components/TaskOrder/TaskOrder.vue

```diff
@@ -29,16 +37,28 @@ import { Component } from "vue-property-decorator";
 import TaskOrderCard from "@/portfolios/portfolio/components/TaskOrder/TaskOrderCard.vue";
 import { TaskOrderCardData } from "../../../../../types/Global";
 import TaskOrderBuildPage from "@/portfolios/portfolio/components/TaskOrder/TaskOrderBuildPage.vue";
```

Comment:
remove import, and delete the actual file `TaskOrderBuildPage.vue` since we won't be using it